hastexo_guacamole_client/guacamole/client.py

```diff
@@ -0,0 +1,221 @@
+"""
+The underlying work for this code is published under the MIT License (MIT)
+(https://pypi.org/project/pyguacamole/)
+"""
+
+import socket
+import logging
+
+from .exceptions import GuacamoleError
+
+from .instruction import INST_TERM
+from .instruction import GuacamoleInstruction as Instruction
+
+# supported protocols
+PROTOCOLS = ('vnc', 'rdp', 'ssh')
+
+PROTOCOL_NAME = 'guacamole'
+
+BUF_LEN = 4096
```

Comment:
I think it wouldn't be terrible to name this `BUFFER_LENGTH`. :) Also, maybe add a comment as to what unit this is. Bytes? KiB? Something else?

Reply:
Fixed in ea3cebaa11390a4b50b70df8cec8e09efc01283b.